pkg/resources/kafka/configmap.go

```diff
@@ -247,17 +245,50 @@ func generateListenerSpecificConfig(l *v1beta1.ListenersConfig, log logr.Logger)
 	if err := config.Set("listeners", listenerConfig); err != nil {
 		log.Error(err, "setting listeners parameter in broker configuration resulted an error")
 	}
-
 	return config
 }
 
+func generateListenerSSLConfig(config *properties.Properties, name string, sSLFormat string, passwordKeyMap map[string]string, log logr.Logger) {
+	if sSLFormat == "JKS" {
+		keyStoreLocKey := fmt.Sprintf("listener.name.%s.ssl.keystore.location", name)
```

Comment:
All these keys and their values could be stored in a `map[string]string` and than loop through the items of the map and invoke `config.Set` for each item.

e.g.:
```
listenerSSLConfig := map[string]string{
  "fmt.Sprintf("listener.name.%s.ssl.truststore.location", name)": namedKeystorePath+"/"+v1alpha1.TLSJKSKeyStore,
...
...
}

for k, v := range listenerSSLConfig {
  if err := config.Set(k, v); err != nil {
 			log.Error(err, fmt.Sprintf("setting %s parameter in broker configuration resulted an error", k))
 		}
}
```

Reply:
Thank you! I have changed it to that one